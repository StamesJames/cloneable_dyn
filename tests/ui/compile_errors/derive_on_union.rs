use cloneable_dyn::CloneDyn;

#[derive(CloneDyn)]
union SomeUnion {
    first: i32,
    second: i8,
}

fn main() {}
