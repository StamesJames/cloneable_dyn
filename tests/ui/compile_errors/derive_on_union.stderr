error: proc-macro derive panicked
 --> tests/ui/compile_errors/derive_on_union.rs:3:10
  |
3 | #[derive(CloneDyn)]
  |          ^^^^^^^^
  |
  = help: message: Unions are not supported
